frontend/src/fsrs_cards.tsx

```diff
@@ -320,6 +321,13 @@ const FSRSCards: React.FC<FSRSCardsProps> = ({
       // Create a new Set and add the guess
       updatedCorrectGuesses = new Set(correctGuesses).add(guess);
       setCorrectGuesses(updatedCorrectGuesses);
+    } else {
+      const guessAlphagram = guess.split("").sort().join("");
```

Comment:
this will work for now. I need to get in some of the alphabet-specific logic from woogles to get the sorting orders correct. For languages like German it's possible the lexicographic sort order is not the same as the alphagram order. For languages with digraphs (which WordVault doesn't support yet) splitting them like this will break it. But we'll fix that later. Thanks for this.